money-api/src/main/java/com/comcast/money/api/SpanFactory.java

```diff
@@ -20,5 +20,9 @@
 
     Span newSpan(String spanName);
 
-    Span newSpan(String spanName, boolean sticky);
+    Span newSpan(SpanId spanId, String spanName);
+
+    Span childSpan(String childName, Span span);
```

Comment:
@clinedome-work haha, that is the way I had it originally, but I felt like there was a growing sophistication in the way spans were created, that is why I moved the behavior from the span to the span factory.

Putting `childSpan` on the Span itself implied that a Span instance had all of the smarts to create a child, and I was uncomfortable with that.